src/main/resources/static/_shared.yml

```diff
@@ -2234,7 +2237,6 @@ components:
           CAS2: '#/components/schemas/SubmitCas2Application'
       required:
         - type
-        - translatedDocument
```

Comment:
i'm not sure why this is optional now?

Reply:
WIP 

Making translated document required results in:

SubmitApprovedPremisesApplication.kt

    @Schema(example = "null", required = true, description = "Any object that conforms to the current JSON schema for an application")
    @get:JsonProperty("translatedDocument", required = true) override val translatedDocument: kotlin.Any?,

And compilation error:
Type of 'translatedDocument' is not a subtype of the overridden property 'public abstract val translatedDocument: Any defined in uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.SubmitApplication'

Line 50 of SubmitApprovedPremisesApplication.kt

SubmitApplication.kt

                @get:Schema(example = "null", requiredMode = Schema.RequiredMode.REQUIRED, description = "Any object that conforms to the current JSON schema for an application")
        val translatedDocument: kotlin.Any

Leaving it as optional results in:

    @Schema(example = "null", description = "Any object that conforms to the current JSON schema for an application")
    @get:JsonProperty("translatedDocument") override val translatedDocument: kotlin.Any? = null

SubmitApplication:

                @get:Schema(example = "null", description = "Any object that conforms to the current JSON schema for an application")
        val translatedDocument: kotlin.Any? 

And no compilation errors.

Currently on main

Reply:
I tried to work around this without luck. Fix for this is going to be merged into main as part of open api generator upgrade

Reply:
Have fixed this on main via https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/2084